src/lib/endpoint.ts

```diff
@@ -99,28 +96,14 @@ export class Endpoint {
     }
 
     const search = await this.client.tm.txSearchAll({ query });
-    const resultsNested = search.txs.map(({ hash, height, result }) => {
```

Comment:
Actually three reasons. I saw the warning _No message.sender nor message.signer in tx …_ a lot of times in the logs, which confuses people trying to get things up and running. The way the code was structured this was also searched in transactions that do not even have a packet send. Then this provides some signer on the transaction basis, but does not tell much about the actual packages. With multiple signers this can be unrelated to the package emission or the packages are sent after a chain of contract executions. And then the field is not used.

Reply:
fair enough